### ISSUE TITLE
Root rights

### ISSUE THREAD

**Issue by Galaska** [juangburgos/WtDesigner#20]
It should be possible to install WtDesigner without giving root to the installer. The destop file could be installed in home.

**Comment by juangburgos** [comment]
You can always build it from source

**Comment by Galaska** [comment]
yes, I could but it still remains a completly unnecessary limitation for those (like me) who just want to quickly try your tool and will never give an installer root rights unless it comes from a well trusted source. The same applies to Windows. No need to install in system directories. You could change that with a realy tiny amount of effort.  
Anyway its nice that you supply such a tool to the community and I might build it later if we come to the conclusion to actually use Wt.

**Comment by bill-auger** [comment]
strike 1
